docs/src/main/paradox/server.md

```diff
@@ -99,13 +99,12 @@ Maven
 ```
 @@@
 
-
-TODO describe settings for code generation: scala/java, client/server/both
+For a complete overview of the configuration options see the chapter for your build tool, @ref[sbt](sbt-plugin.md), Gradle or Maven. (TODO Gradle and Maven)
 
 ## Generate and implement
 
 Define the interfaces you want to implement in your project's
-@sbt[src/main/protobuf]@gradle[src/main/proto] file(s).
+@sbt[src/main/protobuf]@gradle[src/main/proto]@maven[src/main/proto] file(s).
```

Comment:
those are all the same for all build tools tho?

Reply:
I vote for follow the build tool convention